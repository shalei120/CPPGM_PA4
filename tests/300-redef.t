// http://forum.cppgm.org/question/953/additional-tests-for-pa4/?comment=1206#comment-1206
//Coda Highland
#define X(y,y) y
