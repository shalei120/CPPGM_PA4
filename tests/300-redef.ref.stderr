ERROR: duplicate parameter y in macro definition
